protocol/communities/manager.go

```diff
@@ -2381,11 +2381,7 @@ func (m *Manager) shouldUserRetainDeclined(signer *ecdsa.PublicKey, community *C
 	return request.ShouldRetainDeclined(requestClock)
 }
 
-func (m *Manager) HandleCommunityCancelRequestToJoin(signer *ecdsa.PublicKey, request *protobuf.CommunityCancelRequestToJoin) (*RequestToJoin, error) {
-	community, err := m.GetByID(request.CommunityId)
-	if err != nil {
-		return nil, err
-	}
+func (m *Manager) HandleCommunityCancelRequestToJoin(signer *ecdsa.PublicKey, request *protobuf.CommunityCancelRequestToJoin, community *Community) (*RequestToJoin, error) {
 	if community == nil {
 		return nil, ErrOrgNotFound
 	}
```

Comment:
This doesn't make sense anymore, withouth the context of `GetByID`. Should be checked on the caller side of this func.

Reply:
Done